openspace.cfg

```diff
@@ -64,7 +64,7 @@ Asset = "default"
 -- Sets the profile that should be loaded by OpenSpace.  Profiles are going to replace
 -- assets in a future versions and shouldn't be used at the same time as the 'Asset'
 -- setting above
--- Profile = "default"
+Profile = "default"
```

Comment:
This will make OpenSpace master run in profile mode by default!

Reply:
Yea, I left there for now as I had to quickly commit everything when I left for the office this morning.  Discussion pending if we want to make it the default